doc/internal/how_to_release.txt

```diff
@@ -64,11 +64,20 @@ The documentation will be automatically regenerated in the next few hours.
 Generate and upload the package
 ===============================
 
-For release candidates, only upload on PyPI.
+For non-final releases (e.g., release candidates, alpha, beta), upload on PyPI only.
```

Comment:
the "only upload to PyPI" isn't clear if this is for the rest of the document of this section alone.

Can you remove that sentenses and put in the "on mloss.org" section something like:
To this only for final relelase.